### PR TITLE
[EDA] New Lead Fields in EDA Package for Prospect Management

### PR DESCRIPTION
**Pull request by kdy618**
# Changes
- To help your recruiting and admissions team track suspect and prospect data for your institution, we added new fields to the Lead object. For more information review the [EDA Data Dictionary](https://powerofus.force.com/s/article/EDA-Data-Dictionary) and [Lead](https://powerofus.force.com/s/article/EDA-Lead) object reference in the EDA Documentation. 

# Work Items
[W-8987801](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008a0AxIAI/view)

# New Metadata
## Lead Fields: 
- SMS_Opt_Out__c
- Birth_Date__c
- Preferred_Enrollment_Date__c
- GPA__c
- Gender__c
- Ethnicity__c
- Citizenship__c
- Highest_Degree_Earned__c
- Preferred_Enrollment_Status__c
- Preferred_Teaching_Format__c
- Recruitment_Stage__c
- Language__c
- Most_Recent_School__c
- Area_Of_Interest__c
- Undergraduate_Major__c
- External_Id__c
- SSN__c

# Testing Notes
- See https://salesforce.quip.com/0uIcAUI7y0sg

### CODE REVIEW

File: force-app/main/default/lwc/jsconfig.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     },
     "include": [
         "**/*",
-        "../../.sfdx/typings/lwc/**/*.d.ts"
+        "../../.sfdx/typings/lwc/**/*.d.ts",
+        "../../../../.sfdx/typings/lwc/**/*.d.ts"
```
Comment (xinnanlisf) [inline force-app/main/default/lwc/jsconfig.json:26]: Is this on purpose?

Comment (kdy618) [reply force-app/main/default/lwc/jsconfig.json]: @xinnanlisf , nope. Removed this update.

File: force-app/main/default/objectTranslations/Lead-ca/Area_of_Interest__c.fieldTranslation-meta.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <label><!-- Area of Interest --></label>
     <name>Area_of_Interest__c</name>
     <picklistValues>
-        <masterLabel>Undecided</masterLabel>
+        <masterLabel>None</masterLabel>
         <translation><!-- Undecided --></translation>
```
Comment (agoolsby) [inline force-app/main/default/objectTranslations/Lead-ca/Area_of_Interest__c.fieldTranslation-meta.xml]:
```suggestion
        <translation><!-- None --></translation>
```

File: force-app/main/default/objects/Lead/fields/Gender__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Gender__c</fullName>
+    <description>The Lead&apos;s final grade point average.</description>
```
Comment (agoolsby) [inline force-app/main/default/objects/Lead/fields/Gender__c.field-meta.xml:4]:
```suggestion
    <description>The Lead&apos;s self-identified gender.</description>
```

File: force-app/main/default/objects/Lead/fields/SSN__c.field-meta.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>SSN__c</fullName>
```
Comment (agoolsby) [inline force-app/main/default/objects/Lead/fields/SSN__c.field-meta.xml:3]:
```suggestion
    <fullName>SSN__c</fullName>
    <description>The Lead&apos;s Social Security Number. Encrypted for confidentiality. Only users with the View Encrypted Data permission can see the unencrypted value.</description>
```

**Review by agoolsby — CHANGES_REQUESTED** [review]
Some minor changes I caught.

File: force-app/main/default/objects/Lead/fields/Undergraduate_Major__c.field-meta.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Undergraduate_Major__c</fullName>
+    <description>The Lead&apos;s undergraduate degree.</description>
```
Comment (agoolsby) [inline force-app/main/default/objects/Lead/fields/Undergraduate_Major__c.field-meta.xml:4]:
```suggestion
    <description>The primary field of study for the Lead&apos;s undergraduate degree.</description>
```

**Review by agoolsby — DISMISSED** [review]
Label changes from suggestions look good.

File: force-app/main/default/objects/Lead/fields/Birth_date__c.field-meta.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Birth_date__c</fullName>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Lead/fields/Birth_date__c.field-meta.xml:3]: Casing issue. You may have to rename all of the files. Casing changes for file names can be challenging to handle with git. I suggest using git mv
```suggestion
    <fullName>Birth_Date__c</fullName>
```

File: force-app/main/default/objects/Lead/fields/Area_of_Interest__c.field-meta.xml [hunk 7]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Area_of_Interest__c</fullName>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Lead/fields/Area_of_Interest__c.field-meta.xml:3]: Casing issue. You may have to rename all of the files. Casing changes for file names can be challenging to handle with git. I suggest using git mv
```suggestion
    <fullName>Area_Of_Interest__c</fullName>
```

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Only two minor changes on this having to do with casing in the API names of the fields. Unfortunately this may necessitate file name changes.

File: force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 8]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
     <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
-    <label><!-- Area of Interest --></label>
-    <name>Area_of_Interest__c</name>
+    <label>TestArea</label>
```
Comment (agoolsby) [inline force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: Shouldn't this be a comment tag with the English label?

Comment (agoolsby) [reply force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: I see many of the translation file updates have this "test" placeholeder. The other files put the English label in a comment tag. So I'm a bit confused on why we've started using a different approach.

**Review by agoolsby — COMMENTED** [review]
Just some questions about the sample translation string for the updated items. English updates look great!

File: force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml (hunk 8 above)
Comment (MatthewBlanski) [reply force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: @kdy618 Please resolve these issues.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]

File: force-app/main/default/objectTranslations/Lead-ca/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 9]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-ca/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-de/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-de/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-de/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 11]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-de/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 12]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-ca/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-en_GB/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 13]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-en_GB/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-en_GB/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 14]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-en_GB/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-en_US/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 15]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-en_US/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-en_US/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 16]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-en_US/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-es/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 17]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-es/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-es/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 18]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-es/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-es_MX/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 19]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-es_MX/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-es_MX/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 20]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-es_MX/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-fi/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 21]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-fi/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-fi/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 22]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-fi/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-fr/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 23]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-fr/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-fr/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 24]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-fr/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-ja/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 25]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-ja/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-ja/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 26]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-ja/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-nl_NL/Area_Of_Interest__c.fieldTranslation-meta.xml [hunk 27]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s area of interest, such as a program, degree, or concentration. --></help>
+    <label>Area of Interest</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-nl_NL/Area_Of_Interest__c.fieldTranslation-meta.xml:4]: This label should be a stub.

File: force-app/main/default/objectTranslations/Lead-nl_NL/Recruitment_Stage__c.fieldTranslation-meta.xml [hunk 28]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <help><!-- The Lead&apos;s stage in the recruitment process. --></help>
+    <label>Testrecruitment</label>
```
Comment (MatthewBlanski) [inline force-app/main/default/objectTranslations/Lead-nl_NL/Recruitment_Stage__c.fieldTranslation-meta.xml:4]: This label should be a stub.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Please change the marked translation labels to stubs.

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
Passes EDA QA Testing. Merging. Will close EDA WI and inform teams.

**Pull request merged**